tests/utils/test_vcf_handler.py

```diff
@@ -48,7 +54,7 @@ def test_INFOParser_parse_read():
     """
         Test parsing mocked variant entries given different parser info
     """
-    # test multivalue, array of arrays, eg separated with ',' and then  '|'
+    # test multi_value, array of arrays, eg separated with ',' and then  '|'
```

Comment:
I often find it useful to use the Given-When-Then syntax to put the test in context. For instance,
"# Given a multi_value with array of arrays when separated with ','"
- defined data and test

"# Then [descrive expected outcome]"
- assert bla bla